scalafix-tests/input/src/main/scala/test/ImplicitClassPrivateParams.scala

```diff
@@ -0,0 +1,56 @@
+/*
+rules = ImplicitClassPrivateParams
+ */
+package test
+
+object ImplicitClassPrivateParams {
+  implicit class ValueClassXtensionVal(val int: Int) extends AnyVal {
+    def doubled: Int = int + int
+  }
+
+  implicit class ValueClassXtensionAnnotatedVal(@transient val str: String) extends AnyVal {
+    def doubled: String = str + str
+  }
+
+  implicit class XtensionVal(val str: String) {
+    def doubled: String = str + str
+  }
+
+  implicit class XtensionAnnotatedVal(@transient val str: String) {
+    def doubled: String = str + str
+  }
+
+  implicit class XtensionAnnotatedFinalVal(@transient final val str: String) {
+    def doubled: String = str + str
+  }
+
+  implicit class XtensionVar(var str: String) {
+    def doubled: String = str + str
+  }
+
+  implicit class XtensionAnnotatedVar(@transient var str: String) {
+    def doubled: String = str + str
+  }
+
+  implicit class XtensionFinalAnnotatedFinalVar(@transient final var str: String) {
+    def doubled: String = str + str
+  }
+
+  implicit class XtensionAnnotatedProtectedVal(@transient protected val int: Int) {
+    def doubled: Int = int + int
+  }
+
+  implicit class XtensionAnnotatedProtectedVar(@transient protected var int: Int) {
+    def doubled: Int = int + int
+  }
+
+  implicit class Xtension(int: Int) {
```

Comment:
We can ignore non-AnyVals since those fields are private by default unless they're case classes, but case classes are rarely used for implicit classes.